tests/index.test.js

```diff
@@ -14,6 +15,7 @@ dbConnection.once('open', () => {
     .then(() => Place.remove({}))
     .then(() => Event.remove({}))
     .then(() => Product.remove({}))
+    .then(() => Token.remove({}))
```

Comment:
Promise.all ?

Reply:
yeah, this was in the middle of manic debugging so I left it for later, but yeah you're right